### ISSUE TITLE
Add support for copy/paste shortcuts in the System Session Manager terminal

### ISSUE THREAD

**Issue by dennisroche** [akinjide/refined-aws#6]
Add support for copy/paste shortcuts in the System Session Manager in AWS Console, https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-sessions-start.html#start-sys-console

Happy to submit PR if you think it is worthy addition.

**Comment by akinjide** [comment]
Definitely, a good addition but could you provide a use case for context

**Comment by dennisroche** [comment]
Often when I'm working in a SSM session in the browser, I would be copying the results from the terminal to share (e.g. for an incident response, debugging networking); or copying commands from a runbook. 

It would make the experience refined if the user didn't need to right-click to paste/copy and it was bound to ctrl+c/v (or ctrl+shift+c if we want to avoid SIGTERM). SSM already has `reverse-i-search` (ctrl+r) mapped. 

Have you used SSM sessions much?

**Comment by akinjide** [comment]
Sounds good. No, I have no experience with SSM sessions. But we should include this feature and you can definitely pick up this issue.
